CMakeLists.txt

```diff
@@ -1679,6 +1679,7 @@ if(CONFIG_BUILD_OUTPUT_STRIPPED)
             $<TARGET_PROPERTY:bintools,strip_flag_infile>${KERNEL_ELF_NAME}
             $<TARGET_PROPERTY:bintools,strip_flag_outfile>${KERNEL_STRIP_NAME}
             $<TARGET_PROPERTY:bintools,strip_flag_final>
+    COMMAND ${CMAKE_COMMAND} -E sha256sum ${KERNEL_STRIP_NAME}
```

Comment:
i'm sure this is a very useful feature, just like printing of memory usage.
But still, printing of memory usage is user configurable, so why should sha256 checksum be always on ?
(I do note it's tied to strip)
https://github.com/zephyrproject-rtos/zephyr/blob/261025a86ee0b03e3efd5a6279a13489b218a954/Kconfig.zephyr#L462-L463

Should this be a general option that users can enable, so that any output file, `bin`, `hex,` `s19`, etc can have this ?
As a matter of fact, `bin` files are created per default, and a checksum on those would be more safe than using the elf file for checksumming.

Reply:
I agree. "Strip" should not be synonymous with "compute & print checksum" and vice versa.